tracer/test/Datadog.Trace.Tests/Sampling/CustomSamplingRuleRegexTests.cs

```diff
@@ -0,0 +1,132 @@
+// <copyright file="CustomSamplingRuleRegexTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Linq;
+using Datadog.Trace.Sampling;
+using Xunit;
+
+namespace Datadog.Trace.Tests.Sampling
+{
+    [Collection(nameof(Datadog.Trace.Tests.Sampling))]
+    public class CustomSamplingRuleRegexTests
+    {
+        [Fact]
+        public void Constructs_ZeroRateOnly_From_Config_String()
+        {
+            var config = "[{\"sample_rate\":0}]";
```

Comment:
Would it be possible to use """ raw string literals in this file, to make comparing `CustomSamplingRuleRegexTests.cs` and `CustomSamplingRuleGlobTests.cs` easier?

Reply:
Good idea!

Reply:
Done.